hack/common.sh

```
@@ -33,6 +33,13 @@ get_OCP_version() {
   # versions behind OCP Y version
   local DIFFERENCE=5
   local OCP_VER_MINOR=$(($DIFFERENCE+$WMCO_VER_MAJOR))
+  # starting on WMCO 11.y.z, the WMCO major version maps to OCP 5.y
+  if [ "$WMCO_VER_MAJOR" -ge 11 ]; then
+    local OCP_VER_MAJOR=5
+    WMCO_VER_MINOR=$(echo $WMCO_VERSION | cut -d. -f2)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)